### ISSUE TITLE
nil pointer: chainmonitor.go:194

### ISSUE THREAD

**Issue by JFixby** [decred/dcrdata#610]
Current master.

Crash log:
```
2018-08-13 19:08:02.081 [INF] DATD: Log folder:  C:\Users\JCode\AppData\Local\Dcrdata\logs\mainnet
2018-08-13 19:08:02.081 [INF] DATD: Config file: D:\GOWS\dev-s\dcrdata.conf
2018-08-13 19:08:02.081 [INF] DATD: dcrdata version 3.0.0 (Go version go1.10.2)
2018-08-13 19:08:02.081 [INF] DATD: Running in "Lite" mode with only SQLite backend and limited functionality.
2018-08-13 19:08:02.171 [INF] RPCC: Established connection to RPC server localhost:9109
2018-08-13 19:08:02.172 [INF] DATD: Connected to dcrd (JSON-RPC API v3.3.0) on MainNet
2018-08-13 19:08:02.199 [INF] SKDB: Loading ticket pool DB. This may take a minute...
2018-08-13 19:08:02.523 [INF] SKDB: Loading all ticket pool diffs...
2018-08-13 19:08:19.123 [INF] SKDB: Advancing ticket pool DB to tip via diffs...
2018-08-13 19:08:19.506 [INF] SKDB: Pre-populating live ticket cache and computing pool value...
2018-08-13 19:08:42.242 [INF] DATD: SQLite DB successfully opened: dcrdata.sqlt.db
2018-08-13 19:08:42.255 [INF] EXPR: Mean Voting Blocks calculated: 7860
2018-08-13 19:08:42.269 [INF] EXPR: Signals unsupported on windows
2018-08-13 19:08:42.269 [INF] EXPR: Starting WebsocketHub run loop.
2018-08-13 19:08:42.270 [INF] SQLT: Current best block (chain server):     265188
2018-08-13 19:08:42.270 [INF] SQLT: Current best block (sqlite block DB):  260242
2018-08-13 19:08:42.270 [INF] SQLT: Current best block (stakedb):          260242
2018-08-13 19:08:42.295 [INF] SQLT: Scanning blocks 260243 to 261000 (40778 live)...
2018-08-13 19:09:11.933 [INF] SQLT: Scanning blocks 261001 to 262000 (40934 live)...
2018-08-13 19:09:44.154 [INF] SQLT: Scanning blocks 262001 to 263000 (41081 live)...
2018-08-13 19:10:09.566 [INF] SQLT: Scanning blocks 263001 to 264000 (41435 live)...
2018-08-13 19:10:33.455 [INF] SQLT: Scanning blocks 264001 to 265000 (40854 live)...
2018-08-13 19:10:56.788 [INF] SQLT: Scanning blocks 265001 to 265188 (40766 live)...
2018-08-13 19:11:00.897 [INF] SQLT: Rescan finished successfully at height 265188.
2018-08-13 19:11:00.897 [INF] DATD: SQLite sync ended at height 265188
2018-08-13 19:11:00.897 [INF] DATD: PostgreSQL sync ended at height -1
2018-08-13 19:11:00.897 [INF] DATD: All ready, at height 265188.
2018-08-13 19:11:00.901 [WRN] EXPR: Charts are not supported in lite mode!
2018-08-13 19:11:00.911 [INF] EXPR: Signaling new block to 0 websocket clients.
2018-08-13 19:11:02.916 [INF] EXPR: Now serving explorer on http://:8080/
2018-08-13 19:11:02.916 [INF] JAPI: Now serving API on http://:8080/
2018-08-13 19:11:49.686 [INF] SKDB: Connected block 265189 to stake DB.
2018-08-13 19:11:49.689 [INF] BLKD: Block height 265189 connected. Collecting data...
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0xa581d5]

goroutine 270293 [running]:
github.com/decred/dcrdata/db/dcrpg.(*ChainDBRPC).Store(0x0, 0xc05f44aea0, 0xc055f71700, 0x0, 0x0)
	<autogenerated>:1 +0x5
github.com/decred/dcrdata/blockdata.(*chainMonitor).BlockConnectedHandler(0xc053391600)
	D:/GOWS/src/github.com/decred/dcrdata/blockdata/chainmonitor.go:194 +0x8b8
created by main.mainCore
	D:/GOWS/src/github.com/decred/dcrdata/main.go:451 +0x2000
```

**Comment by chappjc** [comment]
Acknowledged.  Thanks for the report.  I see the problem.

**Comment by chappjc** [comment]
Please review and test https://github.com/decred/dcrdata/pull/611

**Comment by chappjc** [comment]
resolved by PR #611
